app_notifier/src/app_notifier/mail_notifier_plugin.py

```diff
@@ -17,6 +17,16 @@ def __init__(self):
 
     def app_manager_start_plugin(self, app, ctx, plugin_args):
         self.start_time = rospy.Time.now()
+        # Set mail title first for extensibility
+        self.use_app_start_time = False
```

Comment:
Member variable should be declared in `__init__()` method?

Reply:
Updated [cfb16ff](https://github.com/knorth55/app_manager_utils/pull/55/commits/cfb16ff7d945da8957af1b9e8bceb37bc5929439)